client/settings/test/settings-section.tsx

```diff
@@ -33,7 +33,7 @@ describe( 'SettingsSection', () => {
 
 	test( 'renders children', () => {
 		render(
-			<SettingsSection title="Foo" description="Bar">
```

Comment:
I removed this because there is no property `description` in the `SettingsSection` component. The correct property name is `Description`